wallet/dapps-communication.md

```diff
@@ -612,12 +612,16 @@ properties:
   amount:
     type: number
     description: Set the amount of MASSA coins given to the block creator.
+  dryRun:
```

Comment:
After reading your definition, I think that this terminology is not clear. I propose to use **Non-Persistent Execution** instead.